generators/rust_generator.js

```diff
@@ -116,18 +116,24 @@ function generateStructDefinition (classes) {
     lifetime +
     ' {' +
     os.EOL +
-    classes.map(generateStructDefinitionItem).join('') +
+    classes.map(generateStructDefinitionItem).join(os.EOL) +
     '}'
   )
 }
 
 function generateStructDefinitionItem (class_) {
-  return '    pub ' + class_.escapedName + ": &'a str," + os.EOL
+  return (
+    generateComment(class_) +
+    '    pub ' +
+    class_.escapedName +
+    ': &\'a str,' +
+    os.EOL
+  )
 }
 
 function generateStructInstance (classes) {
   return (
-    'pub static C: CssClasses = CssClasses {' +
+    'pub const C: CssClasses = CssClasses {' +
```

Comment:
It has to be `static`. I used `const` in the first version but it causes huge app binaries because of inlining.

Reply:
Ok, I dropped that commit.